package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "font.js",
+  "version": "1.0.0",
```

Comment:
This is definitely not the first version of Font.js =D
```suggestion
  "version": "2.0.0",
```

Reply:
Oops! Haha